README.md

```diff
@@ -1,7 +1,32 @@
 ## Lagom Recipes
 
-This repository contains code samples to help understand how to solve common problems. In general, code in each folder of this repository tries to answer a question of type "How do I xyz?".
+This repository contains code samples to help understand how to solve common problems. In general, code in each folder of this repository tries to answer a question of type "How do I _xyz_ ?". For example: "How do I use RDBMS read-sides with Cassandra write-sides?".
+
+Each example is usually built in two steps (git commits) where the first commit introduces a sample project and the second commit introduces the minimum changes required to complete the sample. For example: in commit [9f8de2f](https://github.com/lagom/lagom-recipes/commit/9f8de2f34f0978aeeb0f50cb261345e24da44caf) a new project is added which is edit in commit [b759a18](https://github.com/lagom/lagom-recipes/commit/b759a1821b235603cf65bd1556b57050b76ca69c). You can browse the repo history to find the changes demonstrating how to solve each specific problem.
```

Comment:
For example: commit [9f8de2f](https://github.com/lagom/lagom-recipes/commit/9f8de2f34f0978aeeb0f50cb261345e24da44caf) contains a new project. Commit [b759a18](https://github.com/lagom/lagom-recipes/commit/b759a1821b235603cf65bd1556b57050b76ca69c) contains the edits necessary to complete the example.

Reply:
@rstento I don't get your point here. Should we change the whole paragraph?

This PR is already merged, can you clarify it on the new one that I open. 
https://github.com/lagom/lagom-recipes/pull/14

As such I can adapt this paragraph accordingly. Thanks